RabinKarpAlgorithm/RabinKarp.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RabinKarpAlgorithm
+{
+    public class RabinKarp
+    {
+        int mod = 113;
+        public int GetPatternCount(string baseString, string pattern)
+        {
+            int baseStringLen = baseString.Length;
+            int patternLen = pattern.Length;
+
+            char[] possibleMatchChar = new char[patternLen];
+
+            int patternHash = HashMethod(pattern);
+
+            int cnt = 0;
+            int sum = 0;
+
+            int i, j = 0;
+            int pwr = patternLen - 1;
+
+            bool match = false;
+
+            while (j < patternLen)
```

Comment:
While ti ne treba, pogledaj komentar gore.
Zasto si koristio while a ne for ? U sustini su isti, ali for je mnogo citljiviji.